distros/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine:3.5
+
+LABEL org.label-schema.name="docker-bench-security" \
+      org.label-schema.url="https://dockerbench.com" \
+      org.label-schema.vcs-url="https://github.com/docker/docker-bench-security.git"
+
+RUN \
+  apk add --no-cache \
+    docker \
+    dumb-init \
+    git && \
+  git clone https://github.com/docker/docker-bench-security.git /tmp/bench-security && \
```

Comment:
This `git clone` is unnecessary since  we're building using `docker build -t docker-bench-security .`, and thus you don't need to `cp` and `rm` the scripts.